std_msgs/Header header
sensor_suite/Object[] objects